### PR TITLE
gitg: move libraries to bin

### PR DESCRIPTION
**Pull request by albfan**
Fixes https://github.com/Alexpux/MINGW-packages/issues/4586

### CODE REVIEW

File: mingw-w64-gitg/PKGBUILD [hunk 1]
```diff
@@ -55,6 +55,7 @@ build() {
 package() {
   cd "$srcdir/build-${MINGW_CHOST}"
   DESTDIR=${pkgdir}${MINGW_PREFIX} ninja install
+  mv "${pkgdir}${MINGW_PREFIX}"/lib/*.dll* "${pkgdir}${MINGW_PREFIX}"/bin
```
Comment (mati865) [inline mingw-w64-gitg/PKGBUILD]: Wouldn't this catch `.dll.a` files which have to be inside `/lib`?

Comment (albfan) [reply mingw-w64-gitg/PKGBUILD]: Let me change it. I'm checking libgit2-glib and it has .dll in bin and .dll.a on lib. Why gitg don't do the same without this command

**Comment by albfan** [comment]
gitg.exe doesn't show gitg icon. Am I doing something wrong?

**Comment by mati865** [comment]
I have no experience with Meson so I can't tell for sure but `gitg` build system is not configured correctly for Windows.

**Comment by lazka** [comment]
You need a Windows resource file which points to an .ico file and use "windows.compile_resources" in meson to include it with the .exe (glib uses windows.compile_resources for example to store some metadata in the .exe)

**Comment by albfan** [comment]
@lazka Seems this is the problem. Let's try recovering that config, hopefully I will close this issue

**Comment by albfan** [comment]
@lazka That works https://gitlab.gnome.org/GNOME/gitg/merge_requests/26. Any idea about why libgitg-1.0-0.dll and libgit-ext-1.0-0.dll do not deploy on bin directly? I think it has something to do with shared_library.

About compile_resources, any linux example? I think gitg lacks of it on linux, is something like gnome.compile_resources()? I suppose gitg.rc will not work on linux

**Comment by albfan** [comment]
Seems removing install_dir from shared libraries, meson install correctly, so I will fix gitg master and launch a new release to fix this and other minor bugs, yay!

**Pull request closed without merging**